### ISSUE TITLE
The model cannot handle Chinese prompt correctly

### ISSUE THREAD

**Issue by zkyf** [NVlabs/Sana#137]
Model `Efficient-Large-Model/Sana_1600M_1024px_diffusers` gives an image of a cat but the prompt is asking for an image of a cat.
```
import http.server
import socketserver
import torch
from diffusers import SanaPipeline

PORT = 12345
pipe = SanaPipeline.from_pretrained(
    "Efficient-Large-Model/Sana_1600M_1024px_diffusers",
    variant="fp16",
    torch_dtype=torch.float16,
)
pipe.to("cuda")

pipe.vae.to(torch.bfloat16)
pipe.text_encoder.to(torch.bfloat16)

def generate_img(prompt, img_path, seed):
    image = pipe(
        prompt=prompt,
        height=1024,
        width=1024,
        guidance_scale=5.0,
        num_inference_steps=20,
        generator=torch.Generator(device="cuda").manual_seed(seed),
    )[0]
    image[0].save(img_path)

generate_img("一只白色的狗", "./tmp/一只白色的狗_42.png", 42)
```
Result:
![image](https://github.com/user-attachments/assets/dcb0d3bc-d77d-4dfe-8287-af650a885391)

**Comment by zkyf** [comment]
With seed `1344942119` it gives an image of a woman:
![image](https://github.com/user-attachments/assets/f792bbbd-f2cd-4c96-9eb6-057023ae5d43)

**Comment by lawrence-cj** [comment]
Please use a model with `Multi-Language` tag here to inference Chinese prompt:

https://github.com/NVlabs/Sana/blob/main/asset/docs/model_zoo.md
